README.md

```diff
@@ -221,6 +226,18 @@ The rate limit block specifies the actual rate limit that will be used when ther
 Currently the service supports per second, minute, hour, and day limits. More types of limits may be added in the
 future based on user demand.
 
+### Replaces
+
+The replaces key indicates that this descriptor will replace the configuration set by another descriptor.
+
+If there is a rule being evaluated, and multiple descriptors can apply, the replaces descriptor will drop evaluation of
+the descriptor which it is replacing.
+
+To enable this, any descriptor which should potentially be replaced by another should have a name keyword with a unique
+name in the rate_limit section, and any descriptor which should potentially replace that descriptor should have a name
+keyword in the replaces section in the rate_limit section. Then, whenever limits match to both rules, only the rule
+which replaces the original will take effect, and the limit of the original will not be changed after evaluation.
```

Comment:
Can you add some kind of example to make it more clear on why someone would want to use this? (There is an example below but it's more of a code example, I'm talking about a human readable example.) As a user I'm having a hard time quickly understanding why this is useful.

Reply:
Ok I've improved this.